lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -402,7 +405,12 @@ def _call(self, method: Callable, params: ApiParams, context: ApiContext):
 
         try:
             # perform the Elasticsearch query
-            es_response = method(url, **es_request["kwargs"])
+            es_response = method(
+                url,
+                **es_request["kwargs"],
+                auth=(elastic_user, elastic_password),
+                verify=self.config.get("Indexing", "cert_location"),
+            )
```

Comment:
Again, do we need to handle a fallback to `http` if we're not given a cert or credentials?